### ISSUE TITLE
Rule annotations are null in C# library

### ISSUE THREAD

**Issue by VeraBE** [Azure/PSRule.Rules.Azure#1949]
**Description of the issue**

When running PSRule through the C# library, in `Record(IResultRecord record)` of the `HostContext`, `ruleRecord.Info.Annotations` is null even though the rule has annotations defined like this:

```
{
   "apiVersion":"github.com/microsoft/PSRule/v1",
   "kind":"Rule",
   "metadata":{
      "name":"Azure.Policy.f35b4954cba2",
      "tags":{
         "Azure.Policy/category":"App Service"
      },
      "annotations":{
         "Azure.Policy/id":"/providers/Microsoft.Authorization/policyDefinitions/5bb220d9-2698-4ee4-8404-b9c30c9df609",
         "Azure.Policy/version":"3.0.0"
      }
   },
   "spec":{
      [...]
   }
}

```

**Module in use and version:**

- Module: PSRule.Rules.Azure
- Version: **[e.g. 1.22.0]** 1.23.0-B0009

**Comment by BernieWhite** [comment]
@VeraBE This should be fixed with PSRule v2.7.0-B0070.

**Comment by VeraBE** [comment]
@BernieWhite it is, thank you!
